src/main_app/services/__init__.py

```
@@ -1 +1,93 @@
+    "upsert_template",
+    # template_need_update_service
+    "list_templates_need_update",
+    # jobs_service
+    "delete_job",
+    "create_job",
+    "get_job",
+    "list_jobs",
+    "update_job_status",
+    "cancel_job",
+    "is_job_cancelled",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)